### PR TITLE
Implement the backend, server to database connections and api call

### PR DESCRIPTION
**Pull request by sowmyaneni**
Complete backend portion. Connected server to the database and seeded data to the server.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    db.findOne({id: req.params.id}).exec((err, docs) => {
+        if (err) {
+          res.status(500).send();
+        } else {
+          res.status(200).send(docs);
+        }
+      })
+})
+
+// app.get('/repos', function (req, res) {
```
Comment (erjena) [inline server/index.js]: Do we need this? You might consider adding comments for your commented code, so everyone knows what it for.

File: database/mockData.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
+const faker = require('faker');
+// const mongoose = require('mongoose');
+// mongoose.connect('mongodb://localhost/airbnbInfo', { useNewUrlParser: true });
+const db = require('./index.js');
+
+// let dbSchema = mongoose.Schema({
```
Comment (erjena) [inline database/mockData.js]: Do we need this commented code? You might consider add comments as explanation.

Comment (sowmyaneni) [reply database/mockData.js]: Thanks for the review Erjena! I will commit another one without the comments and concise code.

**Comment by sowmyaneni** [comment]
@erjena: I removed all the commented code in server/index.js and mockData.js files. Let me know if there is anything else, or please merge to master. Thanks!

**Pull request merged**